ui/common/src/commonMain/kotlin/social/androiddev/common/composables/text/MastodonTextField.kt

```diff
@@ -0,0 +1,301 @@
+package social.androiddev.common.composables.text
+
+//import androidx.compose.desktop.ui.tooling.preview.Preview
```

Comment:
I don't think we should make a habit of commenting imports?

Reply:
Unfortunately there is no KMP `@Preview` annotation supported atm. Only desktop. So it must be commented out for all these composables to work with KMP. 

I've been uncommenting it locally to test my previews/composables then commenting out when committing,

https://github.com/JetBrains/compose-jb/issues/1603

Reply:
Are we able to move the annotation package to the preview annotation itself? Like

```
@androidx.compose.desktop.ui.tooling.preview.Preview
fun MyPreview() { ... }
```

This would help localize the preview to *one* spot in your file and if other people follow this pattern we wouldn't have to worry about accidentally forgetting to comment out an import line?

Reply:
I don't think so. Because this is a shared src. So building the android/ios apps will try to build this as well and the `@androidx.compose.desktop.ui.tooling.preview.Preview` is not available, so build will fail

Reply:
Right, still comment it out, just use a fully qualified ref instead of an import so that there's only one block of code to comment.

Reply:
Ah I see what you're saying - yes agreed, I'll make that change moving forward here and elsewhere